oas3.yaml

```diff
@@ -0,0 +1,328 @@
+              status:
+                type: string
+                description: The current status of the service's component.
+                enum:
+                  - Healthy
+                  - Degraded
+                  - Unhealthy
+              description:
+                type: string
+                description: A short description of this component's healthy.
+              data:
+                type: object
+                additionalProperties: true
+                properties: {}
+          readOnly: true
+          xml:
+            name: Status
+      xml:
+        name: Health
+      example:
+        ok: true
+
+    PetV1:
+      required:
+        - name
+        - kind
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: The unique ID of the pet in our database.
+          readOnly: true
+        name:
+          type: string
+          description: The pet's given name, used to call it for cuddles and petting.
+        status:
+          type: string
+          description: The current adoption status of the pet.
+          enum:
+            - available
+            - adopted
+            - unavailable
+        kind:
+          type: string
+          description: The kind of pet, might be a dog, cat or goldfish amongst others.
+          enum:
+            - dog
+            - cat
+            - goldfish
+            - horse
+            - hamster
+        breed:
+          type: string
+          description: The breed of the pet, whether a Golden Retriever or Russian Blue.
+        description:
+          type: string
+          description: A short description of the pet, may use Markdown for rich formatting.
+        birthday:
+          type: string
+          format: date
+          description: The date on which the pet was born, used to determine when treats should be purchased.
+        photos:
+          type: array
+          description: A list of photographs of this pet to help warm hearts and put smiles on faces.
+          items:
+            type: string
+            description: The URL at which the photo of this pet may be downloaded.
+      example:
+        id: "225c5957d7f450baec75a67ede427e9"
+        name: "Fido"
+        status: available
+        kind: dog
+        breed: Labrador
+        description: "Fido is a good boy who loves long walks in the park, playing with his ball and licking faces. He's great with children and an absolute sweetheart."
+        birthday: 2016-04-15
+        photos:
+          - https://upload.wikimedia.org/wikipedia/commons/b/b3/Labrador_on_Quantock_%282307909488%29.jpg
+
+    Error:
+      type: object
+      description: An error describing a problem that the server has encountered or identified.
+      required:
+        - status
+        - traceId
+        - title
+      properties:
+        status:
+          type: number
+          format: integer
+          minimum: 100
```

Comment:
nit: maybe don't include 200 if we respect http semantics

Reply:
Interesting idea, I agree though I'm not sure what an easy way to represent this in JSONSchema would be 🤔 I suppose we could simply say that errors are strictly 4xx and 5xx status codes and be done with it? Let me update to that and see where that gets us.